### PR TITLE
Add check for ESP32 SPI WiFi module into PyBadge test

### PR DESCRIPTION
**Pull request by brentru**

### CODE REVIEW

**Comment by ladyada** [comment]
ok it hink you need to doxygen the new function

**Comment by brentru** [comment]
was missing the `_has_wifi` bool, fixed. letting it travis...

File: Adafruit_Arcada.cpp [hunk 1]
```diff
@@ -111,6 +111,16 @@ bool Adafruit_Arcada::begin(void) {
...
+#ifdef SPIWIFI
+  WiFi.status();
+  delay(100);
+  if (WiFi.status() == WL_NO_MODULE)
+  {
+    _has_wifi = false;
+  } else {
+    _has_wifi = true;
+  }
+#endif
```
Comment (ladyada) [inline Adafruit_Arcada.cpp:125]: _has_wifi should be false if theres no SPIWIFI def

**Review by ladyada — CHANGES_REQUESTED** [review]

File: Adafruit_Arcada.cpp (hunk 1 above)
Comment (brentru) [reply Adafruit_Arcada.cpp:125]: Latest commit sets `_has_wifi` based on if SPIWIFI has been defined.

**Review by ladyada — APPROVED** [review]

**Comment by ladyada** [comment]
ok can merge once travis passes

**Pull request merged**